cardano-cli/test/cardano-cli-golden/Test/Golden/Help.hs

```diff
@@ -97,7 +137,9 @@ hprop_golden_HelpCmds =
     unless isWin32 $ do
       help <-
         filterAnsi
-          <$> execCardanoCLI
+          <$> execCardanoCLIWithEnvVars
+            [ ("CARDANO_CLI_VISIBILITY_LEVEL", "internal")
+            ]
```

Comment:
Could also use the opportunity to tag the files as `internal` or `hidden` (maybe putting them in different folders)